pages/item/_itemId/index.vue

```diff
@@ -1,307 +1,271 @@
+                    :key="`genre-${genre.Id}`"
+                  >
+                    <v-chip
+                      small
+                      link
+                      :class="{ 'ml-2': index > 0 }"
+                      nuxt
+                      :to="`/genre/${genre.Id}?type=${item.Type}`"
+                    >
+                      {{ genre.Name }}
+                    </v-chip>
+                  </v-slide-item>
+                </v-slide-group>
+              </v-col>
+            </v-row>
             <v-row
-              class="my-4 align-center"
-              :class="{
-                'justify-center': !$vuetify.breakpoint.mdAndUp,
-                'ml-1': $vuetify.breakpoint.mdAndUp
-              }"
+              v-if="
+                item && directors.length > 0 && !$vuetify.breakpoint.smAndUp
+              "
+              align="center"
             >
-              <play-button class="mr-2" :item="item" />
-              <item-menu :item="item" outlined :absolute="false" />
-              <like-button :item="item" class="ml-2" />
+              <v-col
+                :cols="twoColsInfoColumn.lCols"
+                :sm="twoColsInfoColumn.lSm"
+                :class="twoColsInfoColumn.lClass"
+              >
+                <label class="text--secondary">{{ $t('directing') }}</label>
+              </v-col>
+              <v-col
+                class="px-0"
+                :cols="twoColsInfoColumn.rCols"
+                :sm="twoColsInfoColumn.rSm"
+              >
+                <v-slide-group>
+                  <v-slide-item
+                    v-for="director in directors"
+                    :key="director.Id"
+                  >
+                    <v-chip small link nuxt :to="`/person/${director.Id}`">{{
+                      director.Name
+                    }}</v-chip>
+                  </v-slide-item>
+                </v-slide-group>
+              </v-col>
             </v-row>
-            <v-col cols="12" md="10">
-              <v-row
-                v-if="item && item.GenreItems && item.GenreItems.length > 0"
-                align="center"
+            <v-row
+              v-if="item && writers.length > 0 && !$vuetify.breakpoint.smAndUp"
+              align="center"
+            >
+              <v-col
+                :cols="twoColsInfoColumn.lCols"
+                :sm="twoColsInfoColumn.lSm"
+                :class="twoColsInfoColumn.lClass"
               >
+                <label class="text--secondary">{{ $t('writing') }}</label>
+              </v-col>
+              <v-col
+                class="px-0"
+                :cols="twoColsInfoColumn.rCols"
+                :sm="twoColsInfoColumn.rSm"
+              >
+                <v-slide-group>
+                  <v-slide-item v-for="writer in writers" :key="writer.Id">
+                    <v-chip small link nuxt :to="`/person/${writer.Id}`">{{
+                      writer.Name
+                    }}</v-chip>
```

Comment:
Unrelated, but there's no way for us to make a rule so this kind of stuff it's linted like this?
```suggestion
                    <v-chip small link nuxt :to="`/person/${writer.Id}`">
                        {{ writer.Name }}
                    </v-chip>
```

Reply:
This annoys me as well. :p. I've just looked into it and couldn't see a rule for it.